demosplan/DemosPlanCoreBundle/ValueObject/KeycloakUserData.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
Strict types declaration is missing

Reply:
[b8801fa](https://github.com/demos-europe/demosplan-core/pull/3284/commits/b8801fa7e86c2fe396cfe65b0c47c2ae993820e7)